spec/controllers/announcements_controller_spec.rb

```diff
@@ -129,5 +149,8 @@
       end
       it_behaves_like 'access denied'
     end
+    context 'GET hide' do
```

Comment:
Now I remember that when we discussed it last time that was the decision made :)
I'll wait for the issue to open